metadatastore/mds.py

```diff
@@ -501,6 +501,15 @@ def find_last(self, num=1):
 
 
 class MDS(MDSRO):
+    def __init__(self, *args, **kwargs):
+        self.__doc__ = super().__init__.__doc__
+        super().__init__(*args, **kwargs)
+        self._INS_METHODS = {'start': self.insert_run_start,
```

Comment:
due to the way that method look up works, this might be better to map to strings and then use `getattr` in `insert`.